specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json

```diff
@@ -0,0 +1,1107 @@
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the resource most related to this alert."
+        }
+      }
+    },
+    "AzureResourceIdentifier": {
+      "type": "object",
+      "description": "Azure resource identifier.",
+      "x-ms-discriminator-value": "AzureResource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceIdentifier"
+        }
+      ],
+      "properties": {
+        "azureResourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "ARM resource identifier for the cloud resource being alerted on"
+        }
+      }
+    },
+    "LogAnalyticsIdentifier": {
+      "type": "object",
+      "description": "Represents a Log Analytics workspace scope identifier.",
+      "x-ms-discriminator-value": "LogAnalytics",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceIdentifier"
+        }
+      ],
+      "properties": {
+        "workspaceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The LogAnalytics workspace id that stores this alert."
+        },
+        "workspaceSubscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The azure subscription id for the LogAnalytics workspace storing this alert.",
+          "pattern": "^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$"
+        },
+        "workspaceResourceGroup": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The azure resource group for the LogAnalytics workspace storing this alert"
+        },
+        "agentId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "(optional) The LogAnalytics agent id reporting the event that this alert is based on."
+        }
+      }
+    },
+    "ResourceIdentifier": {
+      "type": "object",
+      "description": "A resource identifier for an alert which can be used to direct the alert to the right product exposure group (tenant, workspace, subscription etc.).",
+      "discriminator": "type",
+      "required": [
+        "type"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "There can be multiple identifiers of different type per alert, this field specify the identifier type.",
+          "enum": [
+            "AzureResource",
+            "LogAnalytics"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentifierType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "AzureResource"
+              },
+              {
+                "value": "LogAnalytics"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "AlertSimulatorRequestBody": {
+      "type": "object",
+      "description": "Alert Simulator request body.",
+      "properties": {
+        "properties": {
```

Comment:
POST request bodies don't need to have their properties inside a "properties" property.  You can keep it this way or move them to the top level, either is fine.

Reply:
Ok thanks, keeping it this way.